trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -305,42 +460,155 @@ def VLM(conditions,settings,geometry):
     # BFX, BFY, AND BFZ ARE THE COMPONENTS ALONG THE BODY AXES
     # OF THE STRIP FORCE CONTRIBUTION.
     BFX = -  CNC *FSIN + CAXL *FCOS
-    #BFY = - (CNC *FCOS + CAXL *FSIN) *SID
+    BFY = - (CNC *FCOS + CAXL *FSIN) *SID
     BFZ =   (CNC *FCOS + CAXL *FSIN) *COD
 
     # CONVERT CNC FROM CN INTO CNC (COEFF. *CHORD).
-    CHORD_strip = CHORD[LE_ind]
     CNC  = CNC  * CHORD_strip
     BMLE = BMLE * CHORD_strip
 
     # BMX, BMY, AND BMZ ARE THE COMPONENTS ALONG THE BODY AXES
     # OF THE STRIP MOMENT (ABOUT MOM. REF. POINT) CONTRIBUTION.
     X      = VD.XCH[LE_ind]  # These are all LE values
-    #Y      = VD.YCH[LE_ind]  # These are all LE values
+    Y      = VD.YCH[LE_ind]  # These are all LE values
     Z      = VD.ZCH[LE_ind]  # These are all LE values
-    XBAR   = np.ones(sum(LE_ind)) * x_m
-    ZBAR   = np.ones(sum(LE_ind)) * z_m
-    #BMX    = BFZ * Y - BFY * (Z - ZBAR)
+    BMX    = BFZ * Y - BFY * (Z - ZBAR)
+    BMX    = BMX + SICPLE
     BMY    = BMLE * COD + BFX * (Z - ZBAR) - BFZ * (X - XBAR)
-    CDC    = BFZ * SINALF +  BFX * COSALF
+    BMZ    = BMLE * SID - BFX * Y + BFY * (X - XBAR)
+    CDC    = BFZ * SINALF +  (BFX *COPSI + BFY *SINPSI) * COSALF
     CDC    = CDC * CHORD_strip
-
-    ES    = 2*s[0,LE_ind]
-    STRIP = ES *CHORD_strip
-    LIFT  = (BFZ *COSALF - BFX *SINALF)*STRIP
-    DRAG  = CDC*ES 
-
-    MOMENT = STRIP *BMY
-
-    # Now calculate the coefficients for each wing and in total
+    #CMTC   = BMLE + CNC * (0.25 - XLE[LE_ind]) #doesn't affect coefficients, but is in VORLAX
```

Comment:
remove, VORLAX has a bit of spaghetti we don't need to inherit

Reply:
done